addons/website_sale/__manifest__.py

```diff
@@ -33,6 +33,7 @@
     'qweb': ['static/src/xml/*.xml'],
     'installable': True,
     'application': True,
+    'post_init_hook': 'post_init_hook',
```

Comment:
I believe `post_init_hook` is only run when a module is first installed. (I might be wrong)
What happens for users already using v14 with the problem ? I think this will not be called when they update (`odoo-bin -u website_sale`) the module. Do we rely on support to tell them to create the missing filter records ?

Reply:
yes, note that this problem occurs only in the websites created **before** the installation of website_sale. so for these website either website_sale will have to be reinstalled or the website have to be recreated. if you have a better idea on how we can also include the current faulty website in the fix please let me know